### PR TITLE
Parallel query example fixed #846

### LINKED ISSUE #846: Parallel query example is confusing
http://astroquery.readthedocs.io/en/latest/api.html#parallel-queries
```python
from astroquery import module

QC = QueryClass(login_information)

results = parallel_map(QC.query_object,['m31','m51','m17'],radius=['1"','1"','1"'])

results = [QC.query_object_async(obj, radius=r)
    for obj,r in zip(['m31','m51','m17'],['1"','1"','1"'])]
```

While I understand this may be pseudo-code, I still cannot follow it. For example, why is `module` imported but unused? Where do `QueryClass` and `parallel_map` come from?

Is your goal to demonstrate that `parallel_map` and `query_object_async` are interchangeable? Why are there so many ways to accomplish this?

### PR DESCRIPTION
**Pull request by AyushYadav**
@bsipocz  The problem mentioned in the example on the API examples page fixed #846 . Suggestion: adding comments for the module import eg: `from astroquery.eso import Eso` .

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -206,12 +206,13 @@ For multiple parallel queries logged in to the same object, you could do:
...
 
-    from astroquery import module
-
+    from astroquery.module import QueryClass
+    
     QC = QueryClass(login_information)
 
     results = parallel_map(QC.query_object,['m31','m51','m17'],radius=['1"','1"','1"'])
-
+    # Here "parallel_map()" is a parallel implementation of some map function.
```
Comment (bsipocz) [inline docs/api.rst]: I would move this comment out to the narrative part, e.g. into a parenthesis.

Comment (AyushYadav) [inline docs/api.rst]: @bsipocz  Should I put in some example as well for the `import` line?

Comment (AyushYadav) [inline docs/api.rst]: @bsipocz Is this ok?

File: docs/api.rst [hunk 2]
```diff
@@ -206,15 +206,17 @@ For multiple parallel queries logged in to the same object, you could do:
...
+    
     QC = QueryClass(login_information)
 
     results = parallel_map(QC.query_object,['m31','m51','m17'],radius=['1"','1"','1"'])
-
+    
     results = [QC.query_object_async(obj, radius=r)
         for obj,r in zip(['m31','m51','m17'],['1"','1"','1"'])]
 
+Here `parallel_map()` is a parallel implementation of some map function.
```
Comment (pllim) [inline docs/api.rst]: Double backticks here since it is not real, so nothing to link to by Sphinx.

File: docs/api.rst [hunk 3]
```diff
@@ -206,15 +206,17 @@ For multiple parallel queries logged in to the same object, you could do:
 
 .. code-block:: python
 
-    from astroquery import module
-
+    from astroquery.module import QueryClass
+    
     QC = QueryClass(login_information)
 
     results = parallel_map(QC.query_object,['m31','m51','m17'],radius=['1"','1"','1"'])
```
Comment (pllim) [inline docs/api.rst]: While you are at it, would you mind applying some PEP8 to this example? It is a bit hard to read.

**Comment by AyushYadav** [comment]
@pllim Checked and improved according to PEP8 coding style as much as possible for the documentation file. Done in entirety for the code blocks. Let me know if this can be merged or more changes need to be done?

File: docs/api.rst [hunk 4]
```diff
@@ -54,7 +54,8 @@ Query a region around a coordinate.
 
 One of these keywords *must* be specified (no default is assumed)::
 
-    radius - an astropy Quantity object, or a string that can be parsed into one.  e.g., '1 degree' or 1*u.degree.
+    radius - an astropy Quantity object, or a string that can be parsed into one.
+    e.g., '1 degree' or 1*u.degree.
```
Comment (pllim) [inline docs/api.rst]: Needs indentation here.

File: docs/api.rst [hunk 5]
```diff
@@ -206,14 +207,17 @@ For multiple parallel queries logged in to the same object, you could do:
 
 .. code-block:: python
 
-    from astroquery import module
+    from astroquery.module import QueryClass
 
     QC = QueryClass(login_information)
 
-    results = parallel_map(QC.query_object,['m31','m51','m17'],radius=['1"','1"','1"'])
+    results = parallel_map(QC.query_object,['m31','m51','m17'],
```
Comment (pllim) [inline docs/api.rst]: Add a space after each comma.

**Comment by AyushYadav** [comment]
@pllim I used pycodestyle for checking the code part. The space after the comma was not raised and is commonly used so I carried it though. Corrected now. Additions?

**Comment by bsipocz** [comment]
Thanks @AyushYadav.

One useful thing for the future is that you can use ``[skip ci]`` in the commit message to skip running the whole test suite for commits that should have no effect on any of the tests.

**Comment by AyushYadav** [comment]
@bsipocz @pllim Thanks for the reviews. @bsipocz Will keep in mind the advise. Used a feature branch this time, as advised last time 👍

**Pull request merged**